src/test/kotlin/repository/ProjectTableRepoTest.kt

```diff
@@ -61,13 +74,36 @@ class ProjectTableRepoTest : H2DatabaseTest(arrayOf(ProjectTable), true) {
         @Test
         fun `When a project is created, but the assigned user doesn't exist, then an exception is thrown`() =
             testCoroutine {
-                val request =
-                    ProjectOuterClass.Project.Create
-                        .newBuilder()
-                        .setName("Test Project")
-                        .build()
-
+                val request = Project.Create.newBuilder().setName("Test Project").build()
                 assertThrows<NotFoundException.User> { repo.createProject(request, UUID.randomUUID().toString()) }
             }
     }
+
+    @Nested
+    inner class GetAllProjects {
+        @Test
+        fun `When projects are found, then all projects are returned`() = testCoroutine {
+            val project1Id = createExampleProject("Test Project 1", ProjectStatus.PROJECT_STATUS_ACTIVE)
+            val project2Id = createExampleProject("Test Project 2", ProjectStatus.PROJECT_STATUS_ACTIVE_LOCKED)
+
+            val projects = repo.getAllProjects()
+            assertThat(projects).hasSize(2)
+            val firstProject = projects.find { it.id == project1Id }
+            assertThat(firstProject).isNotNull
+            val secondProject = projects.find { it.id == project2Id }
+            assertThat(secondProject).isNotNull
+        }
+
+        @Test
+        fun `When archived and deleted projects exist, then they are not returned`() = testCoroutine {
+            val project1Id = createExampleProject("Test Project 1", ProjectStatus.PROJECT_STATUS_ACTIVE)
+            createExampleProject("Test Project 2", ProjectStatus.PROJECT_STATUS_ARCHIVED)
+            createExampleProject("Test Project 3", ProjectStatus.PROJECT_STATUS_DELETED)
+
+            val projects = repo.getAllProjects()
+            assertThat(projects).hasSize(1)
+            val firstProject = projects.find { it.id == project1Id }
+            assertThat(firstProject).isNotNull
```

Comment:
```suggestion
            val project1Id = createExampleProject("Test Project 1", ProjectStatus.PROJECT_STATUS_ACTIVE)
            val project2Id = createExampleProject("Test Project 2", ProjectStatus.PROJECT_STATUS_ARCHIVED)
            createExampleProject("Test Project 3", ProjectStatus.PROJECT_STATUS_DELETED)

            val projects = repo.getAllProjects()
            assertThat(projects).hasSize(1)
            val firstProject = projects.find { it.id == project1Id }
            assertThat(firstProject).isNotNull
            val secondProject = projects.find { it.id == project2Id }
            assertThat(secondProject).isNull
```
What do you think of this additional check?

Reply:
I like it and added it.